tests/ui/const-generics/defaults/default-param-wf-concrete.next.stderr

```diff
@@ -2,7 +2,7 @@ error[E0080]: attempt to compute `u8::MAX + 1_u8`, which would overflow
   --> $DIR/default-param-wf-concrete.rs:4:28
    |
 LL | struct Foo<const N: u8 = { 255 + 1 }>;
-   |                            ^^^^^^^ evaluation of constant value failed here
+   |                            ^^^^^^^ evaluation of `Foo::{constant#0}` failed here
```

Comment:
This isn't great, ideally it'd say something like "evaluation of unnamed constant inside `Foo` failed here" or so...

Reply:
yea, this may be sth we want to improve for all mentions of unnamed consts. The disambiguators don't help anyone but compiler devs.